tests/test_warning.py

```diff
@@ -7,11 +8,19 @@
 
 
 @pytest.mark.parametrize(
-    "optimization_level, expected_warnings", [(0, 0), (1, 1), (2, 1)]
+    "optimization_level, expected_warnings", [(1, 1), (2, 1)]
 )
-def test_import_warnign(optimization_level, expected_warnings):
-    with pytest.warns(None) as record:
+def test_import_warning(optimization_level, expected_warnings):
+    with pytest.warns() as record:
         with patch("sys.flags", SimpleNamespace(optimize=optimization_level)):
             _check_asserts()
 
     assert len(record) == expected_warnings
+
+
+@pytest.mark.parametrize("optimization_level, expected_warnings", [(0, 0)])
+def test_import_no_warning(optimization_level, expected_warnings):
```

Comment:
```suggestion
@pytest.mark.parametrize("optimization_level", [0])
def test_import_no_warning(optimization_level):
```

Reply:
We don't need the `expected_warnings`, since it is always going to be 0 (and we don't use it in the code).